src/renderer/components/modules/SettingsWindow.tsx

```diff
@@ -70,6 +70,10 @@ const SettingsWindow = (props: {
     });
   }, []);
 
+  const isEmptyApiKey = (apiKey: string): boolean => {
+    return apiKey === "" || apiKey === "null" || apiKey === "undefined";
```

Comment:
Should it be this way: `apiKey === "" || apiKey === null || apiKey === undefined;`?

Reply:
no, the apiKey is force to be of type string, so when missing set it, the null is converted to "null", just see the console

Reply:
You can test it

Reply:
This is TYPE script which make me crazy in my CSE134B Final!

Reply:
But according to [this](https://stackoverflow.com/a/20688019/22896924), `===` suffices.